viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from enum import unique
+
+
 def create_movie(title, genre, rating):
-    pass
+    dict = {}
+    if title and genre and rating:
+        dict['title'] = title
+        dict['genre'] = genre
+        dict['rating'] = rating
+        return dict
+    else:
+        return None

```

Comment:
Let's consider flipping the sense of the condition around to handle the special case (that something is falsy) quickly and get out, rather than having that dangling `else` or return statement at the end. This arrangement is referred to as a guard clause or a guard condition. 


Notice that it lets us move the main focus of the function (creating the new dict) out of an indented block, letting it stand out more clearly.
```suggestion
 if not title or not genre or not rating:
     return None
     
 new_movie = {
     "title" : title,
     "genre" : genre,
     "rating" : rating
 }
 return new_movie
```
It may seem like a silly change or nitpicky, but this is common structuring in Python when using guard clauses.